build/util/update_group_policy.py

```diff
@@ -0,0 +1,62 @@
+#!/usr/bin/env python3
+# Copyright (c) 2024 The Brave Authors. All rights reserved.
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at https://mozilla.org/MPL/2.0/.
+
+import os
+import shutil
+import hashlib
+
+from brave_chromium_utils import wspath
+
+# Chromium stores all group policy definitions under
+# `src/components/policy/resources/templates/policy_definitions/`
+#
+# The name of the file (minus the extension; ex: TorDisable.yaml => TorDisable)
+# corresponds to an auto-generated entry in:
+# `src/out/<build_type_here>/gen/chrome/app/policy/policy_templates.json
+#
+# That auto-generated value (ex: `policy::key::kTorDisabled`) is referenced
+# when we map to a preference in our policy map:
+# `src/brave/browser/policy/brave_simple_policy_map.h`
+#
+# When the code below is ran (trigger via hook; see src/brave/DEPS) this will
+# copy the group policy files from:
+# `src/brave/components/policy/resources/templates/policy_definitions/`
+# to their expected place in Chromium:
+# `src/components/policy/resources/templates/policy_definitions/`
+#
+
+
+def update_policy_files():
+    policy_dir = wspath(
+        "components/policy/resources/templates/policy_definitions/")
```

Comment:
pls use root-based paths to avoid confusion. For the reader it's unclear whether this is relative to the script location or to `//brave`. I assume this should be `//brave/components/...`

Reply:
I didn't know we can use that style of pathing here - I'll give it a shot

Reply:
done